api/v1alpha1/crds.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package v1alpha1
 
-const (
+var (
 	// BlockDeviceResourceKind is the kind of block device CRD
 	BlockDeviceResourceKind = "BlockDevice"
```

Comment:
Except this and `BlockDeviceClaimResourceKind`, all others can be removed as they were used only during CRD installation from the ndm operator which is now removed.